pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyServiceStarter.java

```diff
@@ -72,13 +72,15 @@
 
     @Parameter(names = { "-zk", "--zookeeper-servers" }, description = "Local zookeeper connection string")
     private String zookeeperServers = "";
```

Comment:
This should be `Deprecated` and  add "deprecation note" in description. Or it would be confusing for command line user.

Reply:
@Jason918 done